datadog_checks_base/datadog_checks/base/utils/db/utils.py

```diff
@@ -120,7 +120,7 @@ def acquire(self, key):
 
 def resolve_db_host(db_host):
     agent_hostname = datadog_agent.get_hostname()
-    if not db_host or db_host in {'localhost', '127.0.0.1'}:
+    if not db_host or db_host in {'localhost', '127.0.0.1'} or db_host.startswith('/'):
```

Comment:
What does this change do?

Reply:
It short-circuits the case when a host is specified to be the socket on a local instance. It doesn't actually change any of the functionality; it just returns a bit faster.

Reply:
Got it - thanks!